src/styles/fonts.css

```diff
@@ -0,0 +1,13 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url(../fonts/Avenir-Book.ttf) format('truetype');
+}
+
+@font-face {
+  font-family: Avenir-heavy;
+  font-weight: 600;
+  font-style: normal;
+  src: url(../fonts/Avenir-Heavy.ttf) format('truetype');
```

Comment:
Ensure that the font file `Avenir-Heavy.ttf` exists in the `../fonts/` directory and is accessible.